Extensions/Physics2Behavior/JsExtension.js

```diff
@@ -774,6 +774,11 @@ module.exports = {
       .setDefaultValue('true')
       .getCodeExtraInformation()
       .setFunctionName('setSleepingAllowed');
+    
+    // Deprecated action (fixed typo):
+    aut
+      .addDuplicatedAction("SetSleepingaAllowed", "SetSleepingAllowed")
```

Comment:
Unless I'm mistaken, you should fix the original action name too:

```    
aut
      .addAction(
        'SetSleepingAllowed', 
```

Check the logs in the Chrome debugger console but it's possible that there is a log telling you `Could not find an action with name SetSleepingAllowed`.

Reply:
Oh, I checked! Thank you!